php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S2681.html

```diff
@@ -1,14 +1,19 @@
 <p>Curly braces can be omitted from a one-line block, such as with an <code>if</code> statement or <code>for</code> loop, but doing so can be
 misleading and induce bugs. </p>
-<p>This rule raises an issue when the indentation of the lines after a one-line block indicates an intent to include those lines in the block, but the
-omission of curly braces means the lines will be unconditionally executed once.</p>
+<p>This rule raises an issue when the whitespacing of the lines after a one line block indicates an intent to include those lines in the block, but
+the omission of curly braces means the lines will be unconditionally executed once.</p>
 <h2>Noncompliant Code Example</h2>
 <pre>
 if ($condition)
   firstActionInBlock();
   secondAction();  // Noncompliant; executed unconditionally
 thirdAction();
 
+if($condition) firstActionInBlock(); secondAction();  // Noncompliant; secondAction executed unconditionally
```

Comment:
Does the implementation raise an issue for such a case?

Reply:
No, implementation does not raise an issue. I removed this case from S2681.html